docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -257,8 +257,24 @@ the ESPW.
 These commands administer a deployment of the central components ---
 the kcp server, PKI, and the central KubeStellar components --- in a
 Kubernetes cluster that will be referred to as "the hosting cluster".
-These are framed as "kubectl plugins" and thus need to be explicitly
-or implicitly given a kubeconfig file for the hosting cluster.
+These commands are framed as "kubectl plugins" and thus need to be
```

Comment:
is it only because of the plugins that we need to use kubeconfig of the hosting cluster ?

Reply:
It is the other way around: because these commands need to use a kubeconfig, it is natural to frame them as kubectl plugins.

Reply:
I'm no English expert but got it the other way around .